phpcs.xml.dist

```diff
@@ -71,6 +71,15 @@
             <property name="searchAnnotations" type="boolean" value="true"/>
         </properties>
     </rule>
+    <rule ref="SlevomatCodingStandard.Namespaces.MultipleUsesPerLine.MultipleUsesPerLine">
```

Comment:
can you explain the reason for this (and the other two) change in the coding style rules?

Reply:
As types in DocBlocks may be typed partially only by the Class Name `Product` instead of Namespace + Class Name `SomeNamespace\Product`, we need to resolve them using `use statements`. 
This allows us to cover the case with group use statement, which is not part of PSR standard, however may be used in user land :)